tools/individual-pkg-gen/copy-files-removing-deps.ts

```diff
@@ -34,147 +34,157 @@ function copyFilesRemovingDependencies(): void {
         const esLintRcContents = fs.readFileSync(source);
         fs.outputFileSync(destination, Buffer.concat([esLintRcContents,
           Buffer.from("\nbaseConfig.rules['import/no-extraneous-dependencies'] = ['error', " +
               '{ devDependencies: true, peerDependencies: true } ];\n')]));
       } else if (sourceFileName.endsWith('.ts') && !sourceFileName.endsWith('.d.ts')) {
-        fileProcessed = true;
         const sourceCode = fs.readFileSync(source).toString();
         const sourceCodeOutput = awsCdkMigration.rewriteImports(sourceCode, sourceFileName, {
           customModules: alphaPackages,
         });
         fs.outputFileSync(destination, sourceCodeOutput);
+      } else {
+        const stat = fs.statSync(source);
+        if (stat.isDirectory()) {
+          copyOrTransformFiles(pkg, source, destination);
+        } else {
+          fs.copySync(source, destination);
+        }
       }
+    }
+  }
+}
 
-      if (fileProcessed) {
-        continue;
-      }
+function transformPackageJson(pkg: any, source: string, destination: string, alphaPackages: { [dep: string]: string; }) {
+  const packageJson = fs.readJsonSync(source);
+  const pkgUnscopedName = `${pkg.name.substring('@aws-cdk/'.length)}`;
+
+  packageJson.name = `@aws-cdk-lib-alpha/${pkgUnscopedName}`;
+  packageJson.repository.directory = `packages/individual-packages/${pkgUnscopedName}`;
+
+  // disable awslint (some rules are hard-coded to @aws-cdk/core)
+  packageJson.awslint = {
+    exclude: ['*:*'],
+  };
+
+  // add a pkglint exemption for the 'package name = dir name' rule
+  const pkglint = packageJson.pkglint || {};
+  pkglint.exclude = [
+    ...(pkglint.exclude || []),
+    'naming/package-matches-directory',
+    // the experimental packages need the "real" assert dependency
+    'assert/assert-dependency',
+  ];
+  packageJson.pkglint = pkglint;
+
+  // turn off the L1 generation, which uses @aws-cdk/ modules
+  if (packageJson.scripts?.gen === 'cfn2ts') {
+    delete packageJson.scripts.gen;
+  }
 
-      const stat = fs.statSync(source);
-      if (stat.isDirectory()) {
-        copyOrTransformFiles(pkg, source, destination);
-      } else {
-        fs.copySync(source, destination);
-      }
+  const jsiiTargets = packageJson.jsii.targets;
+  jsiiTargets.dotnet.namespace = jsiiTargets.dotnet.namespace.replace(
+    /^Amazon\.CDK\./, 'Amazon.CDK.Alpha.');
+  jsiiTargets.java.package = jsiiTargets.java.package.replace(
+    /^software\.amazon\.awscdk\./, 'software.amazon.awscdk.alpha.');
+  jsiiTargets.java.maven.artifactId = jsiiTargets.java.maven.artifactId.startsWith('cdk-')
+    ? jsiiTargets.java.maven.artifactId.replace(/cdk-/, 'cdk-alpha.')
+    : 'cdk-alpha.aws-' + jsiiTargets.java.maven.artifactId;
+  jsiiTargets.python.distName = jsiiTargets.python.distName.replace(
+    /^aws-cdk\./, 'aws-cdk.alpha.');
+  jsiiTargets.python.module = jsiiTargets.python.module.replace(
+    /^aws_cdk\./, 'aws_cdk.alpha.');
```

Comment:
This is the only new code in this file - the rest is a refactor. Sourced from https://github.com/aws/aws-cdk/pull/15722#discussion_r678285130